src/components/inputs/button/button.component.tsx

```diff
@@ -59,6 +59,18 @@ export interface ButtonProps {
    * The variant to use.
    */
   variant?: ButtonVariant;
+  /**
+   * The component used for the root node. Either a string to use a HTML element or a component.
+   */
+  component?: any;
+  /**
+   * The accept attribute value is a string that defines the file types the file input should accept.
```

Comment:
Update comment also, its value it's not a string any more

Reply:
Updated to "array of strings"